sdk/loadtestservice/Azure.Developer.LoadTesting/tests/TestV2ClientTest.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Developer.LoadTesting.Models;
+using NUnit.Framework;
+
+namespace Azure.Developer.LoadTesting.Tests
+{
+    public class TestV2ClientTest
+    {
+        [Test]
+        public async Task PathTestAsync()
```

Comment:
Should this be PatchTest?  I just want to make sure I understand the intention of this test and what it is trying to illustrate.